src/services/about.rs

```diff
@@ -16,3 +18,25 @@ pub async fn about(context: &Context<'_>, chain_id: &str) -> ApiResult<About> {
         build_number: build_number(),
     })
 }
+
+pub async fn get_master_copies(
+    context: &Context<'_>,
+    chain_id: &str,
+) -> ApiResult<Vec<MasterCopy>> {
+    let info_provider = DefaultInfoProvider::new(chain_id, &context);
+    let url = core_uri!(info_provider, "/v1/about/master-copies/")?;
+    request_master_copies(&context, url).await
+}
+
+pub async fn request_master_copies(
```

Comment:
Why not make this part of the info provider trait?

Reply:
Just saw the thread above

Reply:
I thought the discussion was still going on here: https://github.com/gnosis/safe-client-gateway/pull/501#discussion_r672156548